src/main/topLevelPages/myDashboard/newVirtualContributorWizard/useNewVirtualContributorWizard.tsx

```diff
@@ -13,25 +13,18 @@ import { LicensePlanType, NewVirtualContributorMySpacesQuery } from '../../../..
 import CreateNewVirtualContributor, { VirtualContributorFromProps } from './CreateNewVirtualContributor';
 import LoadingState from './LoadingState';
 import AddContent from './AddContent';
-import ChooseSubspaceStep1b from './ChooseSubspace.step1b';
+import ExistingSpace, { SelectableKnowledgeProps } from './ExistingSpace';
 import { useTranslation } from 'react-i18next';
 import { useNotification } from '../../../../core/ui/notifications/useNotification';
 import { useUserContext } from '../../../../domain/community/user';
 import DialogWithGrid from '../../../../core/ui/dialog/DialogWithGrid';
+import useNavigate from '../../../../core/routing/useNavigate';
 import { usePlanAvailability } from '../../../../domain/journey/space/createSpace/plansTable/usePlanAvailability';
+import { addVCCreationCache } from './vcCreationUtil';
 
-type Step = 'initial' | 'create_VC' | 'add_knowledge' | 'step1b';
+const SPACE_LABEL = '(space)';
 
-interface SelectedSubspace {
-  id: string;
-  profile: {
-    displayName: string;
-    url: string;
-  };
-  community: {
-    id: string;
-  };
-}
+type Step = 'initial' | 'create_VC' | 'add_knowledge' | 'existingKnowledge';
```

Comment:
Let's agree on a naming convention here

Reply:
yes, let's use camelCase as this is the preferred one for js/react